### PR TITLE
Remove flex layout for facia slices/items at mobile breakpoints

### PR DESCRIPTION
**Pull request by phamann**
As our mobile breakpoint is a single column there is no need to layout with flexbox, which could be potentially affecting rendering of large DOMs for older devices (such as iPhone 4 iPad 1).

This wraps all flex layouts with media queries and ensures certain elements display full width. 

Thanks to @robertberry for the idea! @sndrs if you could have a review that would be great.

### CODE REVIEW

**Comment by robertberry** [comment]
:gem:

**Comment by phamann** [comment]
PR easier reviewed with [`?w=1`](https://github.com/guardian/frontend/pull/8306/files?w=1)

File: static/src/stylesheets/module/facia/_l-list.scss [hunk 1]
```diff
@@ -1,9 +1,11 @@
 .l-list {
     width: 100%;
 
-    .has-flex-wrap & {
-        @include flex-display;
-        @include flex-wrap(wrap);
+    @include mq(mobile) {
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_l-list.scss]: is mobile right?

Comment (phamann) [inline static/src/stylesheets/module/facia/_l-list.scss]: Good catch!

**Comment by sndrs** [comment]
lgtm apart from comment. be good to confirm with a browserstack?

**Comment by phamann** [comment]
Browserstack screenshots here: http://www.browserstack.com/screenshots/d3a757e762099c25d178a3977c800d35e917b3bb

**Comment by phamann** [comment]
@robertberry @sndrs  You both happy for me to merge?

**Comment by robertberry** [comment]
I sure am!

**Comment by sndrs** [comment]
belated :+1:

**Pull request merged**